clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/provider/CloudFoundryProvider.java

```diff
@@ -50,7 +50,7 @@
     singletonMap(new SearchableResource(APPLICATIONS.getNs(), "cloudfoundry"), new ApplicationSearchResultHydrator());
 
   private final Map<String, String> urlMappingTemplates = emptyMap();
-  private final String id = "cloudfoundry";
+  public static final String PROVIDER_ID = "cloudfoundry";
```

Comment:
I believe this is a breaking change somehow, that there are places that rely on `getId()` on the provider. Notice the `@Getter` on the type.

Reply:
If `CloudFoundryProvider` implemented `CloudProvider`, then there might be a problem.  It implements `SearchableProvider`, which does not have that method.